dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.optional.jaxws-2.2.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.internal.optional.jaxws-2.2
+visibility=private
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+Subsystem-Name: Java Web Services API 2.2
+-bundles=\
+ com.ibm.websphere.javaee.jaxws.2.2; require-java:="9"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.xml.ws:jaxws-api:2.2.12"
```

Comment:
since this is an internal optional feature we should drop the `mavenCoordinates` and indicate `apiJar=false`